controller.go

```diff
@@ -595,6 +617,28 @@ func (c *Controller) syncHandler(key string) error {
 				return err
 			}
 
+			if host.Status == provision.ActiveStatus {
```

Comment:
Is this new?

Reply:
@alexellis , no it is in all of the others.

https://github.com/inlets/inlets-operator/blob/e39529b772a7d5da0b67cf39990141010d59e710/controller.go#L649

https://github.com/inlets/inlets-operator/blob/e39529b772a7d5da0b67cf39990141010d59e710/controller.go#L604

https://github.com/inlets/inlets-operator/blob/e39529b772a7d5da0b67cf39990141010d59e710/controller.go#L581